system_tests/common_test.go

```diff
@@ -60,6 +60,182 @@ import (
 type info = *BlockchainTestInfo
 type client = arbutil.L1Interface
 
+type TestClient struct {
+	ctx     context.Context
+	Info    info
```

Comment:
info shouldn't be a prameter here, because it's not client specific. l2info is correct for the entire l1/l2 (it's the list of accounts, their keys/nonce), so there should be 2 infos in builder and none here.